packages/graph-explorer/src/connector/openCypher/queries/fetchSchema.ts

```diff
@@ -140,30 +149,47 @@ const fetchEdgesAttributes = async (
     return edges;
   }
 
-  await Promise.all(labels.map(async labelResult => {
-    const edgesTemplate = edgesSchemaTemplate({
-      type: labelResult
-    });
-
-    const response = await openCypherFetch<RawEdgesSchemaResponse>(edgesTemplate);
-
-    const edge = response.results[0].object as OCEdge;
-    const label = edge["~entityType"] as string;
-    const properties = edge["~properties"];
-    edges.push({
-      type: label,
-      displayLabel: sanitizeText(label),
-      total: countsByLabel[label],
-      attributes: Object.entries(properties || {}).map(([name, prop]) => {
-        const value = prop;
-        return {
-          name,
-          displayLabel: sanitizeText(name),
-          dataType: typeof value === "string" ? "String" : "Number",
-        };
-      }),
-    });
-  }));
+  await Promise.all(
+    labels.map(async labelResult => {
+      const edgesTemplate = edgesSchemaTemplate({
+        type: labelResult,
+      });
+
+      const response = await openCypherFetch<RawEdgesSchemaResponse>(
+        edgesTemplate
+      );
+
+      if (!response.results || response.results.length === 0) {
+        console.warn("Skipping label because there are no edges:", labelResult);
```

Comment:
Is it really a warning or normal behaviour?

Reply:
That's a good question. I actually don't know. How would you like to handle it?

Reply:
I think I am going to go ahead and remove these console logging statements to reduce confusion.